src/content/tutorials/access-management/authentication/automation/index.md

```diff
@@ -0,0 +1,164 @@
+---
+linkTitle: In automation
+title: Authentication for the platform API in automation
+description: Using the platform API from an automation requires a hands-free way to provide credentials to kubectl or any `kubernetes` client. This article explains how to obtain a service account token to use in such a scenario.
+weight: 20
+menu:
+  main:
+    identifier: tutorials-access-management-authentication-automation
+    parent: tutorials-access-management-authentication
+last_review_date: 2024-10-28
+user_questions:
+  - How can I use the platform API in a programmatic way?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-shield
+---
+
```

Comment:
@ssyno  can you read this article and tell me if it is up to date plz?

Reply:
checking

Reply:
LGTM